lib/src/main/java/com/github/okdroid/checkablechipview/CheckableChipView.kt

```diff
@@ -368,7 +371,7 @@ class CheckableChipView @JvmOverloads constructor(
     }
 
     override fun onSaveInstanceState(): Parcelable {
-        return SavedState(super.onSaveInstanceState()).apply {
+        return SavedState(super.onSaveInstanceState()!!).apply {
```

Comment:
Looking at the source `super.onSaveInstanceState()` can actually never return null, looks like a wrong @Nullable annotation

Reply:
[For child classes I guess](https://cs.android.com/android/_/android/platform/frameworks/base/+/517bafa59df45b838245252503b94fc929cf8844...a22a5d83c2ec9d4b0549e92c1d3af7550896e7ee)